solr/core/src/java/org/apache/solr/util/circuitbreaker/CircuitBreaker.java

```diff
@@ -52,4 +58,22 @@ public CircuitBreaker() {}
 
   /** Get error message when the circuit breaker triggers */
   public abstract String getErrorMessage();
+
+  /**
+   * Set the request types for which this circuit breaker should be checked. If not called, the
+   * circuit breaker will be checked for all request types.
```

Comment:
```suggestion
   * circuit breaker will be checked for the `SEARCH` request type only.
```

Reply:
https://github.com/apache/solr/pull/1871/commits/8d30678dc6431974ae1d742f9919b0b8f9134c5c to replace the backtick usage.